packages/hint-typescript-config/src/import-helpers.ts

```diff
@@ -33,7 +33,7 @@ export default class TypeScriptConfigImportHelpers implements IHint {
         const validateTslibInstalled = (evt: ScanEnd) => {
             const { resource } = evt;
 
-            const pathToTslib = path.join(process.cwd(), 'node_modules', 'tslib');
+            const pathToTslib = path.dirname(require.resolve('tslib', { paths: [resource] }));
```

Comment:
If I recall correctly `require.resolve` will throw if `tslib` can't be found, so we should probably move this line inside the `try/catch` block.